src/Symfony/Component/Lock/Store/CombinedStore.php

```diff
@@ -32,11 +32,10 @@ class CombinedStore implements SharedLockStoreInterface, LoggerAwareInterface
     use LoggerAwareTrait;
 
     /** @var PersistingStoreInterface[] */
-    private $stores;
-    /** @var StrategyInterface */
-    private $strategy;
+    private array $stores;
+    private StrategyInterface $strategy;
     /** @var SharedLockStoreInterface[] */
-    private $sharedLockStores;
+    private array $sharedLockStores;
```

Comment:
@jderusse Do you know if this property is in use somehow? I couldn't find any read or write accesses.

Reply:
Backported: #42960